src/index.html

```diff
@@ -11,7 +11,187 @@
+    
+      <fieldset class="form-field">
+        <legend>Registration
+        </legend>
+        <label class="element">
+          E-mail:
+          <input type="email"
+      name="email" 
+      placeholder="email@example.com"
+          >
+        </label>
+      
+        <label class="element">
+            Password:
+          <input type="password"
+       name="password"
+       minlength="5"
+       maxlength="35"
+          > 
+        </label>    
+      </fieldset> 
+     
+      <fieldset class="form-field"> 
+        <legend>An interesting fact about you!
+        </legend>
+        <div class="block">
+          <label>
+              Do you love cats?
+            <label>
+              <input
+              type="radio"
+              name="cats"
+              value="yes"
+              > Yes
+            </label>
+            <label>
+              <input
+              type="radio"
+              name="cats"
+              value="no"
+              > No
+            </label>
+          </label>
+        </div>
+      
+        <label class="element">
+             What is your favorite color?
+          <input type="color"
+            name="color"
+          >
+        
+        </label>
+    
+        <label class="element">
+             What time do you go to bed?
+          <input type="time"
+             name="time"
+             step="1"
+          > 
+         
+        </label>
+        <label class="element">
+              What are your favorites brand of cars? 
+          <select name="cars" 
+           multiple
+          >
+            
+            <option value="bmw">
+              BMW
+            </option>
+            <option value="audi">
+              Audi
+            </option>
+            <option value="lada">
+              Lada
+            </option>
+          </select>
+        </label>
+
+        <label class="element">
+          How do you rate our work?
+          <input 
+           type="range"
+            name="work"
+            min="0"
+            max="5"
+            step="1"
+            value="0"
+          >
+        </label>
+      </fieldset>
+   
+      <fieldset>
+        <legend>Additional info</legend> 
+        <label class="element"> 
+              Comments:
+          <textarea 
+              name="comment">
+
+          </textarea>
+        </label> 
+
+        <label class="element">
+              Would you recommend us?
+          <select
+              name="recommend">
+            <option value="yes">
+               Yes
+            </option>
+            <option value="no">
+              No
+            </option> 
+            <option value="maybe">
+              maybe
+            </option>
+          </select>
+        </label>
+      </fieldset>
+      <button
+      type="submit"
+      id="button-submit"
+      >
+        Submit
+      </button>
+    </form>
+
+  
+
+
+    
     <script type="text/javascript" src="./main.js"></script>
```

Comment:
```suggestion
    </form>
    <script type="text/javascript" src="./main.js"></script>
```
why do you need this empty lines?